symengine/calculate.h

```diff
@@ -0,0 +1,30 @@
+#ifndef SYMENGINE_CALCULATE_H
+#define SYMENGINE_CALCULATE_H
+
+#include <symengine/visitor.h>
+#include <symengine/basic.h>
+#include <symengine/assumptions.h>
+
+namespace SymEngine
+{
+
+RCP<const Basic> gamma_multiple_2(const RCP<const Basic> &arg);
```

Comment:
We don't need to export these two functions right? We can even wrap them in an anonymous namespace in the .cpp file to keep number of symbols in the binary low I think?